lib/linux/background-managers/cinnamon.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const {commandExists, execFile, grep} = require('../util');
+
+exports.isAvailable = async () => {
+	if (!await commandExists('gsettings')) {
+		return false;
+	}
+
+	try {
+		const {stdout} = await execFile('gsettings', ['list-schemas']);
+
+		return grep(stdout, 'org.cinnamon.desktop.background');
+	} catch (_) {}
+};
+
+exports.set = async function (imagePath) {
```

Comment:
Use arrow functions whenever possible. This applies to many other places in the code too.